site/public/js/docker_interface.js

```diff
@@ -17,3 +17,122 @@ function collapseSection(id,btn_id) {
         btn.innerHTML = 'Collapse';
     }
 }
+
+function filterOnClick() {
+    const this_filter = $(this).data('capability');
+
+    $('.filter-buttons').each(function (){
+        $(this).addClass('fully-transparent');
+    });
+
+    if ($(this).hasClass('fully-transparent')) {
+        $(this).removeClass('fully-transparent');
+    }
```

Comment:
Do we need the `if` here, or can we just always run `$(this).removeClass('fully-transparent');` given that the result is that all elements that enter this function will not have `full-transparent` class.